Quadriga/src/main/resources/locale/errormessages.properties

```diff
@@ -1,6 +1,7 @@
 error_message_401 = Unauthorized Access. Please check login credentials.
-error_message_403 = Not authorized to access this page.
+error_message_403 = Not authorized to access this page or the server is unable to full fill the request.

```

Comment:
You're not authorized to access this page or the server is unable to fulfill the request. 

When is that thrown? This are two different http status, and should not be put into one answer.